syntax = "proto3";

package mypackage;

service MyService {
  rpc GetMessage (MessageRequest) returns (MessageResponse);
}

message MessageRequest {
  string name = 1;
}

message MessageResponse {
  string message = 1;
}
